### PR TITLE
add cassiopee release 1.0.9

### PR DESCRIPTION
**Pull request by osallou**
* [X ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by druvus** [comment]
@osallou Shouldn't the name be cassiopee-c? They also have a ruby version that is called cassiopee

**Comment by osallou** [comment]
the ruby version is not packaged and "deprecated"
and cassiopee is the name already used in bioconda packages (this is only a version upgrade)

File: recipes/cassiopee/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 package:
   name: cassiopee
-  version: "1.0.5"
+  version: "1.0.9"
 
 build:
-  number: 2
+  number: 1
```
Comment (druvus) [inline recipes/cassiopee/meta.yaml]: Should start at 0 for new version.

File: recipes/cassiopee/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 source:
-  url: https://github.com/osallou/cassiopee-c/archive/1.0.5.tar.gz
-  md5: 38f07e5d2ee2a55679281bce73f9993e
+  url: https://github.com/osallou/cassiopee-c/archive/1.0.9.tar.gz
+  md5: b2a9ba2641b3fb2dff9aa6c09b796f2d
```
Comment (druvus) [inline recipes/cassiopee/meta.yaml]: Recommended to use sha256

**Pull request merged**